S3/Crypto.py

```diff
@@ -346,3 +348,11 @@ def calculateChecksum(buffer, mfile, offset, chunk_size, send_chunk):
 
     return md5_hash.hexdigest()
 __all__.append("calculateChecksum")
+
+def sha256_hash_to_base64(sha256_hash):
+    # Extract digest from sha256 hash
+    sha256_hash_digest = sha256_hash.digest()
+    # Then convert it to base64
+    sha256_hash_digest_b64 = b64encode(sha256_hash_digest).decode()
```

Comment:
In this module encodestring (which imported above as an alias of encodebytes when available) is the function used for base64 encoding.
I think you can use:
```python
sha256_hash_digest_b64 = encodestring(sha256_hash_digest).strip().decode()
```
to align with other usecases.